pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1991,7 +1987,7 @@
   tutorial_codesterssj_longdescription: "In this activity, students use our drag-to-text toolkit to start coding in Python right away! As they work through introductory skill building activities, students begin to learn about basic syntax, variables, and events! They'll use multiple sprites to set up a joke telling scenario that they can share with their friends! *Electronic sharing requires student sign up."
   tutorial_cmchat_longdescription: "Join a robot monkey in an hour-long course where you will learn the basics of chatbot programing. In the activity, you will advance through a 15 exercise-long course in which you will learn how to program your very own trivia chatbot."
   tutorial_tynkerdb_longdescription: "Choose a dragon, then embark on a quest for treasure. Blast through obstacles and gain cool power-ups as you journey through a mystical jungle. Learn sequencing, loops, conditional logic, and debugging skills along the way."
-  tutorial_cmbuild_longdescription: "Take your coding skills to the next level by learning to build your own games. In this course, you will learn keyboard user-interface and game mechanics as you build a Super Mario™-styled game. Afterwards, you will be able to share your game!" 
+  tutorial_cmbuild_longdescription: "Take your coding skills to the next level by learning to build your own games. In this course, you will learn keyboard user-interface and game mechanics as you build a Super Mario™-styled game. Afterwards, you will be able to share your game!"
```

Comment:
intentional?

Reply:
Yes.  White space was deleted after the removal of three strings that are not being used.